rsync_to_rdisc.py

```diff
@@ -33,18 +33,20 @@ def send_email(subject, template, body_params, attachments=None):
     )
 
 
-def send_mail_lost_mount(run_file):
+def send_mail_lost_mount(mount_name, run_file):
     hostname = gethostname()
     send_email(
-        f"ERROR: mount lost to BGarray for {hostname}", "lost_mount.html",
-        {"hostname": hostname, "run_file": run_file},
+        subject=f"ERROR: mount lost to {mount_name} for {hostname}",
+        template="lost_mount.html",
+        body_params={"mount_name": mount_name, "hostname": hostname, "run_file": run_file},
```

Comment:
I do prefer the variable assignment, as you propose here. Much easier to read. 
However I though you would prefer positional assignment, right? I could be confused between argument parsing and variable assignment in function calls.

Anyway... any guidelines when to use positional inputs without variable assignment? Only if input is concise?

Reply:
I am not following any hard rules, typically I try to check whether it would improve readability and interpretation of the code if I use named variables.